<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="pnlGame" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>521</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#BBB;</string>
    </property>
    <widget class="QWidget" name="pnlBounce" native="true">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>10</y>
       <width>16</width>
       <height>70</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>671</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#222;</string>
     </property>
    </widget>
    <widget class="QWidget" name="pnlBall" native="true">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>480</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>671</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:red;</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>16</width>
      <height>521</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>671</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maximum">
     <number>450</number>
    </property>
    <property name="singleStep">
     <number>70</number>
    </property>
    <property name="pageStep">
     <number>70</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>671</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Spiel Starten</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPoints">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>671</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Punkte: 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
